pkg/api/v1/project_teams.go

```diff
@@ -27,14 +27,15 @@ type Team struct {
 	Roles []TeamRole `json:"roles"`
 }
 
-func (in *Team) ToAtlas(teamID string) *mongodbatlas.ProjectTeam {
-	result := &mongodbatlas.ProjectTeam{
-		TeamID:    teamID,
-		RoleNames: make([]string, 0, len(in.Roles)),
+func (in *Team) ToAtlas(teamID string) admin.TeamRole {
+	roleNames := make([]string, 0, len(in.Roles))
+	result := admin.TeamRole{
+		TeamId:    &teamID,
+		RoleNames: &roleNames,
 	}
 
 	for _, role := range in.Roles {
-		result.RoleNames = append(result.RoleNames, string(role))
+		*result.RoleNames = append(*result.RoleNames, string(role))
```

Comment:
use getters and setters to avoid pointer operations https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_5_best_practices.md#using-getters-instead-of-direct-field-access